libs/utils/utils.js

```diff
@@ -793,7 +790,10 @@ export async function loadIms() {
       },
       onError: reject,
     };
-    loadScript(`${base}/deps/imslib.min.js`);
+    const path = PAGE_URL.searchParams.get('useAlternateImsDomain')
+      ? 'https://auth.services.adobe.com/imslib/imslib.min.js'
```

Comment:
I'm guessing there's no need to consider stage URL, right?

Reply:
Don't think so 🤔 We didn't have anything before at least

Reply:
We did have an use-case for loading STAGE, but I think we can skip it now.